web/src/components/ChatController.tsx

```diff
@@ -37,12 +37,18 @@ const ChatController = ({ city, language }: ChatControllerProps): ReactElement =
   const isBrowserTabActive = useIsTabActive()
 
   useEffect(() => {
-    if (isBrowserTabActive) {
-      const pollMessageInterval = setInterval(refreshMessages, POLLING_INTERVAL)
+    const messageCount = chatMessages?.messages.length ?? 0
+    const RefreshMethod = () => {
+      if (isBrowserTabActive) {
+        refreshMessages()
+      }
+    }
+    if (messageCount > 0) {
+      const pollMessageInterval = setInterval(RefreshMethod, POLLING_INTERVAL)
```

Comment:
🤔 Why not just
```suggestion
    const messageCount = chatMessages?.messages.length ?? 0
    if (isBrowserTabActive && messageCount > 0) {
      const pollMessageInterval = setInterval(refreshMessages, POLLING_INTERVAL)
```

Reply:
If the interval got triggered you can't stop 'refreshMessages()' from running when isBrowserTabActive is not active 🤔

Unless we clear the interval with return...